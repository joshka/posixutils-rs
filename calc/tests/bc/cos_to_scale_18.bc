a = c(0.5)
b = c(1)
c = c(1.5)
d = c(2)
e = c(2.5)
f = c(3)

scale = 18
a/1
b/1
c/1
d/1
e/1
f/1
quit
