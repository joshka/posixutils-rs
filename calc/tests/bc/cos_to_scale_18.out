0.877582561890372716
0.540302305868139717
0.070737201667702910
-0.416146836547142386
-0.801143615546933714
-0.989992496600445457
